catalyst/data/dataset.py

```diff
@@ -187,7 +190,7 @@ def __init__(
                 parameters.
         """
         list_data = [
-            {"features": filename, "targets": label_fn(filename)}
+            {features_key: filename, "targets": label_fn(filename)}
```

Comment:
could we also parametrise "targets" as `targets_key`? what do you think about it?